modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/i18n/en_US.ts

```diff
@@ -93,6 +93,7 @@ export default {
 	'component': 'Component',
 	'components': 'Components',
 	'confirm-password': 'Confirm Password',
+	'copy': 'Copy',
```

Comment:
Let's use that way, because we can combine in a more fashion way, in our code

```suggestion
	'copy-x': 'Copy - {0}',
```